sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/UnaryPrefixOperatorRepeated.vb

```diff
@@ -0,0 +1,27 @@
+﻿Imports System
+Imports System.Collections.Generic
+
+Namespace Tests.Diagnostics
+    Class UnaryPrefixOperatorRepeated
+        Private Shared Sub NonComp(ByVal bbb As Boolean)
```

Comment:
same here, `bbb` can be removed

Reply:
`bbb` is used a few lines below (the same method is called recursively)